button.js

```diff
@@ -2,7 +2,7 @@ function changeColor() {
   var blueDiv = document.getElementById("changeable");
   // Use the Mozilla Developer Network resource to find the right way to change
```

Comment:
You should remove all comments in code unless they serve a purpose

Reply:
i removed them now